### DISCUSSION TITLE
Constant Mining

### DISCUSSION THREAD

**Discussion by ezbrzys** [RainbowMiner/RainbowMiner#2111] [Ideas]
I have RainbowMiner set to only mine to NiceHash. Is there a way for it to consistently mine instead of doing a "run" then waiting 56 seconds and then doing another run?

I understand the point of this when it's checking to see if there's a more profitable pool, but, since I'm just mining to the one pool, it doesn't seem necessary (unless I am misunderstanding something, which I may very well be).

Also I'm running it on a machine running Ubuntu, and, if the dev is reading this, you're doing a phenomenal job. This is by far the best algo/pool-switching miner I've ever used on any operating system, and the only way I could get my linux machine to mine to NiceHash.

**Comment by RainbowMiner** [comment]
> Also I'm running it on a machine running Ubuntu, and, if the dev is reading this, you're doing a phenomenal job. This is by far the best algo/pool-switching miner I've ever used on any operating system, and the only way I could get my linux machine to mine to NiceHash.

Thank you so much!!

> I have RainbowMiner set to only mine to NiceHash. Is there a way for it to consistently mine instead of doing a "run" then waiting 56 seconds and then doing another run?

The loop is to evaluate the most profitable pool and algorithm - so even if you are on Nicehash only, it makes sense to check for the most profitable algorithm. But - you can just increase the loop interval: just change the parameter `Interval` in config.txt (or use the setup page on localhost:4000). It's in seconds. Don't make it too high though, try 300 (->5 Minutes) or 600 (->10 Minutes), for example. Anyway, with a higher interval, you might miss short profitability spikes.

**Reply by ezbrzys** [reply]
Perfect! Thank you!!!
